File: LC/Python/Easy/2566.py
class Solution:
    def minMaxDifference(self, num: int) -> int:
        max_num = str(num)
        min_num = str(num)

        for c in max_num:
            if c != '9':
                max_num = max_num.replace(c, "9")
                break
        for c in min_num:
            if c != '0':
                min_num = min_num.replace(c, "0")
                break
        return int(max_num) - int(min_num)

File: LC/C++/Easy/2566.cpp
class Solution {
public:
    int minMaxDifference(int num) {
        string max_num = to_string(num);
        string min_num = max_num;

        for (char c : max_num) {
            if (c != '9') {
                replace(max_num.begin(), max_num.end(), c, '9');
                break;
            }
        }
        
        char c = min_num[0];
        replace(min_num.begin(), min_num.end(), c, '0');

        return stoi(max_num) - stoi(min_num);
    }
};